build-system/babel-config/pre-closure-config.js

```diff
@@ -46,6 +46,17 @@ function getPreClosureConfig() {
       },
     },
   ];
+  const experimentConstant = getExperimentConstant();
+  // For FixedLayer experiment
+  if (experimentConstant === 'MOVE_FIXED_LAYER') {
+    // For experiment = true, remove import from viewport-impl.js
+    filterImportsPlugin[1]['imports']['./../fixed-layer'] = ['FixedLayer'];
+  } else {
+    // For experiment = false, remove import from amp-viewer-integration.js
+    filterImportsPlugin[1]['imports']['../../../src/service/fixed-layer'] = [
+      'FixedLayer',
+    ];
+  }
```

Comment:
maintainability nit: Instead of mutating `filterImportsPlugin` via a hardcoded array index, can try rewriting this logic as follows? (I believe a variable object key can be used in ES6 code.)

```js
  const fixedLayerImport =
    getExperimentConstant() == 'MOVE_FIXED_LAYER'
      ? './../fixed-layer'
      : '../../../src/service/fixed-layer';
  const filterImportsPlugin = [
    'filter-imports',
    {
      imports: {
        // Imports removed for all ESM builds.
        './polyfills/document-contains': ['installDocContains'],
        './polyfills/domtokenlist': ['installDOMTokenList'],
        './polyfills/fetch': ['installFetch'],
        './polyfills/math-sign': ['installMathSign'],
        './polyfills/object-assign': ['installObjectAssign'],
        './polyfills/object-values': ['installObjectValues'],
        './polyfills/promise': ['installPromise'],
        './polyfills/array-includes': ['installArrayIncludes'],
        './ie-media-bug': ['ieMediaCheckAndFix'],
        '../third_party/css-escape/css-escape': ['cssEscape'],
        [fixedLayerImport]: ['FixedLayer'],
        // Imports that are not needed for valid transformed documents.
        '../build/ampshared.css': ['cssText', 'ampSharedCss'],
        '../build/ampdoc.css': ['cssText', 'ampDocCss'],
      },
    },
  ];
```